### PR TITLE
[TAS-2374]💄 Update specTableItem label style for campaign item

### PR DESCRIPTION
**Pull request by AuroraHuang22**
截圖 2024-10-08 晚上9 42 12
截圖 2024-10-08 晚上9 43 00

### CODE REVIEW

File: src/components/NFTBook/SpecTableItemAvailableFormat.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <component :is="tag">
     <NFTBookSpecTableLabel :text="$t('nft_details_page_label_content_types')" />
-    <div class="font-[600]">{{ label }}</div>
+    <div :class="['font-[600]', labelStyle]">{{ label }}</div>
```
Comment (nwingt) [inline src/components/NFTBook/SpecTableItemAvailableFormat.vue:4]: Maybe create a new component `<NFTBookSpecTableValue />` for this div

Comment (AuroraHuang22) [reply src/components/NFTBook/SpecTableItemAvailableFormat.vue]: [💄 Add <NFTBookSpecTableValue/>](https://github.com/likecoin/liker-land/pull/1883/commits/fea2e8c7df386da746997b8125d100f6751584ae)

File: src/components/NFTBook/SpecTableItemAccessMethod.vue [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <NFTBookSpecTableLabel
       :text="$t('nft_details_page_label_access_methods')"
     />
-    <div class="font-[600]">{{ label }}</div>
+    <div :class="['font-[600]', labelStyle]">{{ label }}</div>
```
Comment (nwingt) [inline src/components/NFTBook/SpecTableItemAccessMethod.vue:6]: What I mean is that `<NFTBookSpecTableValue />` also replaces this

this component accepts `preset` type e.g. 'campaign'

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/components/NFTBook/SpecTableItemAccessMethod.vue (hunk 2 above)
Comment (AuroraHuang22) [reply src/components/NFTBook/SpecTableItemAccessMethod.vue]: [♻️ Refactor with <NFTBookSpecTableValue/>](https://github.com/likecoin/liker-land/pull/1883/commits/72644913263bd43ce5f2c751d3894e163e9d2119)

**Review by nwingt — APPROVED** [review]

**Pull request merged**